src/main/java/utility/Ui.java

```diff
@@ -0,0 +1,123 @@
+package utility;
+
+import tasktypes.Task;
+
+import java.util.ArrayList;
+
+/**
+ * The {@code Ui} class handles the user interface messages, displaying
+ * various responses to the user based on their interactions with the application.
+ */
+public class Ui {
+    
+    /**
+     * Displays the welcome message to the user when the application starts.
+     */
+    public void welcomeMessage() {
```

Comment:
```suggestion
    public void showWelcomeMessage() {
```
`welcomeMessage` might not be very descriptive of the action this message is going to perform. Perhaps, adding a `show` or `display` prefix might better describe the method.